content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md

```diff
@@ -192,21 +197,21 @@ All code snippets in Windows sections are to be run in a PowerShell environment
     The "pause" (infrastructure) image is hosted on Microsoft Container Registry (MCR). You can access it using "docker pull mcr.microsoft.com/k8s/core/pause:1.2.0". The DOCKERFILE is available at https://github.com/kubernetes-sigs/sig-windows-tools/tree/master/cmd/wincat.
     {{< /note >}}
 
-1. Prepare a Windows directory for Kubernetes
+3. Prepare a Windows directory for Kubernetes
```

Comment:
i think the numbering should be `1.` for all entries actually, because there are problems with the markdown parser.